open-banking-brasil-financial-api-1_ID1.md

```diff
@@ -220,58 +219,45 @@ In addition, the Authorization Server
+O Servidor de Autorização deve suportar as disposições especificadas na cláusula 5.2.2.1 de [Financial-grade API Security Profile 1.0 - Parte 2: Advanced] [FAPI-1-Advanced]
 
-In addition, if the `response_type` value `code id_token` is used, the Authorization Server
+Além disso, se o valor `response_type` `code id_token` for usado, o servidor de autorização:
 
-1. should not return sensitive PII in the ID Token in the authorization response, but if it needs to,
-then it shall encrypt the ID Token.
+1. não deve retornar Informação de Identificação Pessoal (PII) confidenciais no token de ID na resposta de autorização, mas se for necessário,
+então ele deve criptografar o token de ID.
 
-#### Requesting the "cpf" Claim
+#### Solicitando a reivindicação "cpf"
 
-This profile defines "cpf" as a new standard claim as per
- clause 5.1 [OIDC]
+Este perfil define "cpf" como uma nova reivindicação padrão de acordo com cláusula 5.1 [OIDC]
 
-The **CPF** number (Cadastro de Pessoas Físicas, [sepeˈɛfi]; Portuguese for "Natural Persons Register")
- is the **Brazilian** individual taxpayer registry identification. This number is attributed by
- the **Brazilian** Federal Revenue to Brazilians and resident aliens who, directly or indirectly,
-  pay taxes in **Brazil**.
-In the Brasil Open Banking identity model, the cpf is a string consisting of numbers that is 11
-characters long and may start with a 0.
-If the cpf Claim is requested as an Essential Claim for the ID Token or UserInfo response with a
-values parameter requesting a specific cpf value, the Authorization Server MUST return an cpf Claim Value
-that matches the requested value. If this is an Essential Claim and the requirement cannot be met,
- then the Authorization Server MUST treat that outcome as a failed authentication attempt.
+O número do **CPF** (Cadastro de Pessoas Físicas, [sepeˈɛfi]; português para "Registro de Pessoas Físicas") é o cadastro de pessoa física **brasileira**. Este número é atribuído pela Receita Federal **Brasileira** para brasileiros e estrangeiros residentes que, direta ou indiretamente, pagar impostos no **Brasil**.
 
-Name: cpf, Type: String, Regex: '^\d{11}$'
+No modelo de identidade do Open Banking Brasil, o cpf é uma string composta por números 11 caracteres de comprimento e podem começar com 0.
+Se a reivindicação cpf for solicitada como uma reivindicação essencial para o token de ID ou resposta UserInfo com um parâmetros de valores solicitando um valor cpf específico, o Servidor de Autorização DEVE retornar um valor de reivindicação cpf que corresponde ao valor solicitado. Se esta for uma reivindicação essencial e o requisito não puder ser atendido,
+  então, o Authorization Server DEVE tratar esse resultado como uma tentativa de autenticação falhada.
 
-#### Requesting the "cnpj" Claim
+Nome: cpf, Tipo: String, Regex: '^\d{11}$'
 
-This profile defines "cnpj" as a new standard claim as per
- clause 5.1[OIDC]
+#### Solicitando a reivindicação "cnpj"
```

Comment:
As I said before, I suggest to remove this section either.

Reply:
the communication from the security working group was to continue to define CPF and CNPJ as the ecosystem consistent OIDC claims to facilitate data ownership correlation between banks and as such both the claims and mechanisms for support them are required.

Removing this section would be a normative change and will need to go into Implementers Draft 2 as Banks and TPPs have been told to expect these claims and the conformance harness will require their presence in order to pass.